src/confighttp.cpp

```diff
@@ -309,6 +309,13 @@ void saveApp(resp_https_t response, req_https_t request) {
     pt::read_json(ss, inputTree);
     pt::read_json(config::stream.file_apps, fileTree);
 
+    // Moonlight checks the id of an item to determine if an item was changed
+    // Add a property named "id" to the inputTree with a value of the current timestamp
+    // Time is in seconds because some Moonlight clients cannot accept more than a 32-bit integer
+    // Milliseconds would be a better option
+    time_t id = time(nullptr);
```

Comment:
I've changed this to just be a random number given the upcoming API. I didn't add any guard logic here because it seems unnecessary. Let me know what you think

Reply:
> I've changed this to just be a random number given the upcoming API. I didn't add any guard logic here because it seems unnecessary. Let me know what you think

My fear with using a random number is it's possible for them to repeat. I've seen cases where random functions are not really very random at all.

Here's a potential idea. Use an incrementing counter and store the counter as a separate property in the `apps.json`. Then just use that value and add 1 to it when assigning a new id?

Reply:
Given the small chance of collision with this generator vs potential bugs with something like a user-writable counter in a threaded context I would probably lean towards the former?

I could switch to a more precise timestamp with some truncation/casting but this felt a little safer to me

Reply:
How about random then, but check if the id is already used? I feel if we allow it to duplicate, it definitely will happen. I'm used to the SQL auto incrementing style that eliminates the chance of duplication.